A82-xds-system-root-certs.md

```diff
@@ -0,0 +1,81 @@
+A82: xDS System Root Certificates
+----
+* Author(s): @markdroth
+* Approver: @ejona86, @dfawley
+* Status: {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2024-07-08
+* Discussion at: https://groups.google.com/g/grpc-io/c/BgqeUU0q4fU
+
+## Abstract
+
+We will add a new xDS option to use the system's default root
+certificates for TLS certificate validation.
+
+## Background
+
+Most service mesh workloads use mTLS, as described in [gRFC A29][A29].
+However, there are cases where it is useful for applications to use
+normal TLS rather than using certificates for workload identity, such as
+when a mesh wants to move some workloads behind a reverse proxy.
+
+gRPC already has code to find the system root certificates on various
+platforms.  However, there is currently no way for the xDS control plane
+to tell the client to use that functionality, at least not without the
+cumbersome setup of duplicating that functionality in a certificate
+provider config in the xDS bootstrap file.
+
+### Related Proposals: 
+* [gRFC A29: xDS mTLS Security][A29]
+
+[A29]: A29-xds-tls-security.md
+
+## Proposal
+
+We have added a [`system_root_certs`
+field](https://github.com/envoyproxy/envoy/blob/84d8fdd11e78013cd50596fa3b704e152512455e/api/envoy/extensions/transport_sockets/tls/v3/common.proto#L399)
+to the xDS `CertificateValidationContext` message (see
+envoyproxy/envoy#34235).  If this field is present and the
+`ca_certificate_provider_instance` field is unset in the gRPC client,
```

Comment:
"in the gRPC client" is awkward. "for a" would be better. Dunno what is best.

Reply:
I've attempted to clarify the wording here.